### ISSUE TITLE
Consolidate duplicate tc-loader scripts

### ISSUE THREAD

**Issue by tohojo** [xdp-project/bpf-examples#9]
We should probably consolidate the different copies of the TC loader scripts in subdirectories into a single version in a parent/lib directory...

**Comment by simosund** [comment]
Same thing probably goes for the [configure](https://github.com/xdp-project/bpf-examples/blob/master/traffic-pacing-edt/configure) script to detect if iproute has libbpf support.

**Comment by tohojo** [comment]
Yeah. IIRC, @netoptimizer expressed a preference for keeping separate configure scripts in each subdir, but they can still share code :)

**Comment by simosund** [comment]
Worth mentioning that in my current [PR](https://github.com/xdp-project/bpf-examples/pull/13) I've extended the tc loader script somewhat to support attaching a pinned program (and gotten rid of the configure script).

**Comment by netoptimizer** [comment]
As libbpf now support TC-attaching directly, we should consider converting "pping" and "traffic-pacing-edt" to use that instead of mentioned shell script.

**Comment by simosund** [comment]
Yep, converting pping to use libbpf for attaching the tc program is on my TODO list, so this issue is likely goning to resolve itself that way.
